### DISCUSSION TITLE
Button Link  _chevronCustom (navigation)

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1597] [Proposals]
### Goal

Espelhar o conceito aprovado com o componente que temos em Mística.

### Context

Atualmente, a propriedade "navigation" não está espelhada com o conceito que foi aprovado no Novo App. 

Precisamos deixar como disponibilizado na POC abaixo, tanto o tamanho, quanto o alinhamento. 

### Impact

Ao adicionar o chevronCustom corretamente no componente Button Link, os Designers não vão precisar dar detach no componente para fazer o ajuste manualmente .

### Discussion Questions

ND

### Do you have a visual proposal?

https://www.figma.com/file/YFI1cuoCEE3qzNBwQu2ADY/Button-Link---Icon-Chevron?type=design&node-id=1%3A86&mode=design&t=HEuqwTkyUkRQ11n0-1

**Comment by yceballost** [comment]
We worked on it months ago ([ticket](https://jira.tid.es/browse/OBVIVO-1758)). In the design and implementation process we had agreements with your team about how the chevron would be implemented, and this is how is implemented today.

We can reevaluate this need in the future to reach an extra level of customization

**Reply by annefranca** [reply]
Teria uma previsão de data para olhar para essa demanda? Pergunto porque preciso passar essa devolutiva para gerencia da Vivo.
